modules/seqexec/server/src/test/scala/seqexec/server/SeqexecServerLensesSpec.scala

```diff
@@ -13,13 +13,19 @@ import seqexec.server.SeqexecServerArbitraries._
 import gem.arb.ArbObservation
 import gem.Observation
 import monocle.law.discipline.LensTests
+import seqexec.engine.{Action, Actions, Result}
 
 /**
   * Tests SeqexecServer Lenses
   */
 final class SeqexecServerLensesSpec extends CatsSuite with ArbObservation {
 
-  implicit val steppEq: Eq[HeaderExtraData => engine.Step[IO]] = Eq.fromUniversalEquals
+  // I tried to go down the rabbit hole with the Eqs, but it is not worth it for what they are used.
+  implicit val streamEq: Eq[fs2.Stream[IO, Result]] = Eq.fromUniversalEquals
```

Comment:
This doesn't look right. You shouldn't be able to compare `Stream[IO, N]` I think